src/main/scala/beam/sim/BeamMobsim.scala

```diff
@@ -123,6 +113,11 @@ class BeamMobsim @Inject()(
     eventsManager.initProcessing()
     val iteration = actorSystem.actorOf(
       Props(new Actor with ActorLogging {
+        override val supervisorStrategy: SupervisorStrategy =
+          OneForOneStrategy(maxNrOfRetries = 1, withinTimeRange = 1 minute) {
```

Comment:
You can remove the `withinTimeRange` since retries is 1:

If set it is https://github.com/akka/akka/blob/master/akka-actor/src/main/scala/akka/actor/FaultHandling.scala#L65 and take it away and you get https://github.com/akka/akka/blob/master/akka-actor/src/main/scala/akka/actor/FaultHandling.scala#L44 - which is the same thing.